### PR TITLE
Updated Russian translation for current develop branch

### PR DESCRIPTION
**Pull request by Alexell**
Verified on the release v3.3.0

### CODE REVIEW

**Comment by TimZ99** [comment]
I guess that you didn't change the commit since I reviewed #612. Could you have a look at that review?

**Review by TimZ99 — CHANGES_REQUESTED** [review]
See #612

**Comment by Alexell** [comment]
In the #612 PR there was an extra commit that you do not need to add to your repository, so I recreated it.

**Comment by TimZ99** [comment]
I noticed, I only reviewed the translation, not the Telegram part :)

**Comment by Alexell** [comment]
@TimZ99 What do I need to do to add an updated translation?
Or should other participants approve the changes?

**Review by Alexell — COMMENTED** [review]
Here are all correct. The file was re-created based on the updated en_US.lang.php

File: src/lang/ru_RU.lang.php [hunk 1]
```diff
@@ -124,10 +138,11 @@
 		'email' => 'E-mail',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php:138]: Does E-mail have a translation or is it just E-mail?

File: src/lang/ru_RU.lang.php [hunk 2]
```diff
@@ -210,74 +251,91 @@
...
+		'pushover_error_noapp' => 'Не удалось отправить тестовое уведомление: Pushover "App API token" не найден в основных настройках.',
+		'pushover_error_nokey' => 'Не удалось отправить тестовое уведомление: Pushover ключ не найден в вашем профиле.',
+		'telegram_sent' => 'Уведомление в Telegram отправлено',
+		'telegram_error' => 'Произошла ошибка при отправке уведомления в Telegram: %s',
+		'telegram_error_notoken' => 'Не удалось отправить тестовое уведомление: Telegram API token не найден в основных настройках.',
+		'telegram_error_noid' => 'Не удалось отправить тестовое уведомление: идентификатор чата не найден в вашем профиле.',
 		'log_retention_period' => 'Период хранения логов',
 		'log_retention_period_description' => 'Количество дней хранения логов уведомлений и архива аптайма серверов. Введите 0 для выключения очистки логов.',
-		'log_retention_days' => 'дней',
+		'log_retention_days' => 'days',
```
Comment (TimZ99) [inline src/lang/ru_RU.lang.php]: Why is it translated back into English?

**Review by TimZ99 — CHANGES_REQUESTED** [review]
I fixed the other point. Only those two point need some attention.

File: src/lang/ru_RU.lang.php (hunk 1 above)
Comment (Alexell) [reply src/lang/ru_RU.lang.php:138]: Translation is available, but in our country it is most often used simply **E-Mail** and everyone understands what it is about.

File: src/lang/ru_RU.lang.php (hunk 2 above)
Comment (Alexell) [reply src/lang/ru_RU.lang.php]: Oops. My mistake.

Comment (TimZ99) [reply src/lang/ru_RU.lang.php]: Should I change it back?

Comment (Alexell) [reply src/lang/ru_RU.lang.php]: @TimZ99 yes:)

File: docs/credits.rst [hunk 3]
```diff
@@ -153,6 +153,7 @@ The following people have contributed to the translation of PHP Server Monitor:
 
   * Roman Beylin - https://github.com/roman-beylin
   * Yuriy Lyutov - https://github.com/delysh
+  * Alexell - https://github.com/Alexell
```
Comment (TimZ99) [inline docs/credits.rst:156]: If you want something else here, now is the time to say it :P

Comment (Alexell) [reply docs/credits.rst:156]: @TimZ99 It's okay, just fine. Thank you)

**Review by TimZ99 — APPROVED** [review]

**Comment by TimZ99** [comment]
Thank you very much!

**Pull request merged**